### ISSUE TITLE
Version 1.0.3 not working

### ISSUE THREAD

**Issue by H3xtyy** [McMackety/velocity-queue#6]
Like in the title.

```
[23:07:53] [main/ERROR]: Can't create plugin velocityqueue
com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, kotlin.UninitializedPropertyAccessException: lateinit property config has not been initialized
  at org.mcmackety.vqueue.QueuePlugin.<init>(QueuePlugin.kt:28)
  at com.velocitypowered.proxy.plugin.loader.java.VelocityPluginModule.configure(VelocityPluginModule.java:31)
  while locating org.mcmackety.vqueue.QueuePlugin

1 error
	at com.google.inject.internal.InternalProvisionException.toProvisionException(InternalProvisionException.java:226) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1097) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1131) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.velocitypowered.proxy.plugin.loader.java.JavaPluginLoader.createPlugin(JavaPluginLoader.java:104) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.velocitypowered.proxy.plugin.VelocityPluginManager.loadPlugins(VelocityPluginManager.java:135) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.velocitypowered.proxy.VelocityServer.loadPlugins(VelocityServer.java:268) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.velocitypowered.proxy.VelocityServer.start(VelocityServer.java:210) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.velocitypowered.proxy.Velocity.main(Velocity.java:55) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property config has not been initialized
	at org.mcmackety.vqueue.QueuePlugin.createQueue(QueuePlugin.kt:252) ~[?:?]
	at org.mcmackety.vqueue.QueuePlugin.<init>(QueuePlugin.kt:41) ~[?:?]
	at org.mcmackety.vqueue.QueuePlugin$$FastClassByGuice$$f0e28011.newInstance(<generated>) ~[?:?]
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:114) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:91) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:306) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:168) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:39) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	at com.google.inject.internal.InjectorImpl$1.get(InjectorImpl.java:1094) ~[velocity.jar:1.1.4-SNAPSHOT (git-1f9c2456-b350)]
	... 6 more
```

**Comment by McMackety** [comment]
Alright, give me half an hour to see this.

**Comment by McMackety** [comment]
Just fixed it, will update the 1.0.3 release with the new version once built.

**Comment by McMackety** [comment]
Just updated it, try redownloading version 1.0.3
